app/views/aisles/index.html.erb

```diff
@@ -1,22 +1,19 @@
 <% content_for(:title, 'Aisles') %>
 
 <h1>
-  <%= link_to "", new_aisle_path, class: Icon.new %>
+  <%= link_to MaterialIcon.new(icon: :plus_circle, title: 'Create new Aisle', classes: 'float-right').render, new_aisle_path %>
   Aisles
 </h1>
 
 <% if current_user.aisles.any? %>
-
   <% @aisles.each do |aisle| %>
-    <h3 class='index-item controls-on-right-parent'>
-      <%= aisle.order_number %> | <%= aisle.name %> <span style="font-size: 1rem;"></span>
+    <p class='index-item controls-on-right-parent'>
+      <%= aisle.order_number %> | <%= aisle.name %>
       <span class='controls'>
-        <%= link_to "", aisle, method: :delete, data: { confirm: 'Are you sure? This action cannot be undone.' }, class: Icon.delete %>
```

Comment:
Moved the `delete` action into the edit form so it is not accidentally deleted from the index page while on mobile.